app/presenters/step_nav_presenter.rb

```diff
@@ -6,8 +6,9 @@ def initialize(step_nav)
 
   def render_for_publishing_api(publish_intent = PublishIntent.minor_update)
     payload = required_fields
-    payload.merge!(optional_fields)
-    payload.merge(publish_intent.present)
+    payload
+      .merge(optional_fields)
```

Comment:
I think you need the `!` on merge, or it will be overwritten by the publish_intent.

Reply:
I've removed the change as it wasn't relevant to what I'm doing here 👍